app/views/accounts/_connected_app.html.slim

```diff
@@ -0,0 +1,10 @@
+.p2.clearfix.border-top
+  .clearfix.mxn1
+    .sm-col.sm-col-6.px1
+      .bold.truncate
+        - if identity.return_to_sp_url.present?
+          = link_to(identity.display_name, identity.return_to_sp_url)
+        - else
+          = identity.display_name
+    .sm-col.sm-col-6.px1.sm-right-align
+      = t('account.connected_apps.associated') + ' ' + identity.created_at_in_words
```

Comment:
This is probably okay, but I'd be a lot more comfortable with the translations if we passed the creation date/time into the translation as a parameter.